lib/manageiq/providers/workflows/engine.rb

```diff
@@ -24,6 +24,7 @@ def self.init_loggers
 
           require "floe"
           Floe.logger = $workflows_log
+          Floe::Workflow::Runner.docker_runner = floe_docker_runner
```

Comment:
NOTE this obviously shouldn't live in `init_loggers` but until we have another method like maybe just `.init_plugin` this will do for testing.

Reply:
You can create an initializer the "normal" way via config/initializers.  Downside, is it would be run inside all instances.

I'd be ok with an init_plugin method, but I'm not sure how it would be different from a config/initializer.

Reply:
interesting idea to set the values in the `config/initializers` (at install time)
rather than detecting the values.